autogtp/Management.cpp

```diff
@@ -269,6 +269,8 @@ QString Management::getOptionsString(const QJsonObject &opt, const QString &rnd)
     if (rnd != "") {
         options.append(" -s " + rnd + " ");
     }
+    // Additional options specified by server
+    options.append(getOption(opt, "other_options", "", ""));
```

Comment:
What is this for?

Reply:
Can be used to give arbitrary command line options to leelaz. For example timemanage, randomvisits, randomtemp. However I just removed it from this pull request since now non_pruning timemanagement is enabled by default.